### PR TITLE
Update version change for MM 4.3.2

### PR DESCRIPTION
**Pull request by minhtuev**
1. Update the version change for MM 4.3.2
2. Remove the code example for Whatsapp integration - we are pointing to the repo file instead

### CODE REVIEW

File: source/versions/history.rst [hunk 1]
```diff
@@ -1,6 +1,47 @@
 Package History
 ===============
 
+4.3.2 (2020-10-15)
+------------------
+
+Major Features and Improvements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Add auto entity annotating tool which leverages language models like BERT to add entity annotations to training queries
```
Comment (vijay120) [inline source/versions/history.rst:10]: Does spacy use BERT?

File: source/versions/history.rst [hunk 2]
```diff
@@ -1,6 +1,47 @@
...
+
+- Allow the MindMeld tokenizer to preserve special characters
+
+- Allow the MindMeld app to configure the max history length that they should keep
+
+- Allow the role classifier to process a single label
+
+- Expose ES score in QA responses
+
+- Implement the progress bars for classifier training
```
Comment (vijay120) [inline source/versions/history.rst:26]:
```suggestion
- Add progress bars for classifier training
```

File: source/versions/history.rst [hunk 3]
```diff
@@ -1,6 +1,47 @@
...
+------------------
+
+Major Features and Improvements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Add auto entity annotating tool which leverages language models like BERT to add entity annotations to training queries
+
+- Add model tracking through DVC integration
+
+- Add sentiment feature for classifiers
```
Comment (vijay120) [inline source/versions/history.rst:14]:
```suggestion
- Add sentiment features for classifiers
```

File: source/versions/history.rst [hunk 4]
```diff
@@ -1,6 +1,47 @@
...
+
+- Allow the MindMeld app to configure the max history length that they should keep
+
+- Allow the role classifier to process a single label
+
+- Expose ES score in QA responses
+
+- Implement the progress bars for classifier training
+
+- Surface up values from Duckling
```
Comment (vijay120) [inline source/versions/history.rst:28]:
```suggestion
- Allow detailed entity resolution from Duckling
```

File: source/versions/history.rst [hunk 5]
```diff
@@ -1,6 +1,47 @@
...
+
+- Add support for custom resolution in custom evaluation function for Automatic Slotfilling
+
+- Allow the MindMeld tokenizer to preserve special characters
+
+- Allow the MindMeld app to configure the max history length that they should keep
+
+- Allow the role classifier to process a single label
+
+- Expose ES score in QA responses
```
Comment (vijay120) [inline source/versions/history.rst:24]:
```suggestion
- Expose Elasticsearch scoring in QA responses
```

File: source/versions/history.rst [hunk 6]
```diff
@@ -1,6 +1,47 @@
...
+
+- Implement the progress bars for classifier training
+
+- Surface up values from Duckling
+
+
+Bug fixes
+^^^^^^^^^
+
+- Request entities should be an immutable map
```
Comment (vijay120) [inline source/versions/history.rst:34]:
```suggestion
- Fix issue were entities were not immutable in the request object
```

File: source/versions/history.rst [hunk 7]
```diff
@@ -1,6 +1,47 @@
...
+
+- Don't try to load system entity map if it is not there
```
Comment (vijay120) [inline source/versions/history.rst:36]:
```suggestion
- Fixed issue were the system entity recognizer would be loaded without being initialized
```

File: source/versions/history.rst [hunk 8]
```diff
@@ -1,6 +1,47 @@
...
+
+
+Legacy
+^^^^^^
+
+- Add log warnings for Python 3.5; we will remove official support in the next release
```
Comment (vijay120) [inline source/versions/history.rst:42]:
```suggestion
- Add log warnings for Python 3.5; we will officially remove support in the next release
```

**Review by vijay120 — COMMENTED** [review]
Few comments

File: source/versions/history.rst (hunk 1 above)
Comment (ktick) [reply source/versions/history.rst:10]: I don't think so. Even if it does, it'd be better to keep this more generic and just say "Add auto entity annotating tool which leverages pre-trained NLP models to add entity annotations to training queries"

**Review by ktick — COMMENTED** [review]
Looks good to merge once @vijay120's comments are addressed.

File: source/versions/history.rst (hunk 1 above)
Comment (minhtuev) [reply source/versions/history.rst:10]: Done

File: source/versions/history.rst (hunk 3 above)
Comment (minhtuev) [reply source/versions/history.rst:14]: Done

**Review by vijay120 — APPROVED** [review]

**Pull request merged**